Cargo.toml

```
@@ -31,12 +31,12 @@ rand = { version = "0.8", default-features = false, features = ["std_rng"] }
 rstest = "0.13"
 
 [features]
-default = ["fuel-asm/default", "fuel-crypto/default", "fuel-merkle?/default", "fuel-types/default", "std"]
-alloc = ["fuel-merkle", "fuel-types/alloc", "itertools/use_alloc", "serde/alloc"]
+default = ["fuel-asm/default", "fuel-crypto/default", "fuel-merkle/default", "fuel-types/default", "std"]
+alloc = ["fuel-types/alloc", "itertools/use_alloc", "serde/alloc"]
 builder = ["alloc", "internals"]
 internals = []
 random = ["fuel-crypto/random", "fuel-types/random", "rand"]
-std = ["alloc", "fuel-asm/std", "fuel-crypto/std", "fuel-merkle?/std", "fuel-types/std", "itertools/default", "rand/default", "serde?/default"]
+std = ["alloc", "fuel-asm/std", "fuel-crypto/std", "fuel-merkle/std", "fuel-types/std", "itertools/default", "rand/default", "serde?/default"]
```

rand should be guarded by the `random` flag, although it's being auto-included whenever std is enabled.

```suggestion
std = ["alloc", "fuel-asm/std", "fuel-crypto/std", "fuel-merkle/std", "fuel-types/std", "itertools/default", "rand?/default", "serde?/default"]
```